_data/examples.json

```diff
@@ -217,6 +217,10 @@
     {
       "name": "interactive_splom",
       "title": "Scatterplot Matrix"
+    },
+    {
+      "name": "concat_layer.vl.json",
+      "title": "Interactive Dashboard"
```

Comment:
Do you have a name for the upper plot?

"Coordinated bar chart and XXX"? 

(Interactive Dashboard seems pretty generic?)

Reply:
Cross highlight